src/DssSpell.sol

```diff
@@ -75,334 +60,216 @@ contract DssSpellAction is DssAction {
-    address internal immutable MCD_VAT                     = DssExecLib.vat();
-    address internal immutable MCD_VOW                     = DssExecLib.vow();
-    address internal immutable MCD_GOV                     = DssExecLib.mkr();
-    address internal immutable MIP21_LIQUIDATION_ORACLE    = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
-    address internal immutable RWA007_A_URN                = DssExecLib.getChangelogAddress("RWA007_A_URN");
-    address internal immutable RWA014_A_URN                = DssExecLib.getChangelogAddress("RWA014_A_URN");
-    address internal immutable PIP_MKR                     = DssExecLib.getChangelogAddress("PIP_MKR");
-    address internal immutable VOTE_DELEGATE_PROXY_FACTORY = DssExecLib.getChangelogAddress("VOTE_DELEGATE_PROXY_FACTORY");
-    address internal immutable MCD_SPLIT                   = DssExecLib.getChangelogAddress("MCD_SPLIT");
-    address internal immutable USDS_JOIN                   = DssExecLib.getChangelogAddress("USDS_JOIN");
-    address internal immutable USDS                        = DssExecLib.getChangelogAddress("USDS");
-    address internal immutable MKR_SKY                     = DssExecLib.getChangelogAddress("MKR_SKY");
-    address internal immutable SKY                         = DssExecLib.getChangelogAddress("SKY");
-    address internal constant NEW_PIP_MKR                  = 0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b;
-    address internal constant VOTE_DELEGATE_FACTORY        = 0xC3D809E87A2C9da4F6d98fECea9135d834d6F5A0;
-    address internal constant REWARDS_LSMKR_USDS           = 0x92282235a39bE957fF1f37619fD22A9aE5507CB1;
-    address internal constant LOCKSTAKE_MKR                = 0xb4e0e45e142101dC3Ed768bac219fC35EDBED295;
-    address internal constant LOCKSTAKE_ENGINE             = 0x2b16C07D5fD5cC701a0a871eae2aad6DA5fc8f12;
-    address internal constant LOCKSTAKE_CLIP               = 0xA85621D35cAf9Cf5C146D2376Ce553D7B78A6239;
-    address internal constant LOCKSTAKE_CLIP_CALC          = 0xf13cF3b39823CcfaE6C2354dA56416C80768474e;
-
-    // ---------- Wallets ----------
-    address internal constant AAVE_V3_TREASURY   = 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c;
-    address internal constant EARLY_BIRD_REWARDS = 0x14D98650d46BF7679BBD05D4f615A1547C87Bf68;
+    uint256 internal constant FIVE_PT_TWO_PCT_RATE = 1000000001607468111246255079;
+
+    // --- Math ---
+    uint256 internal constant RAD = 10 ** 45;
+
+    address internal immutable USDS                    = DssExecLib.getChangelogAddress("USDS");
+    address internal immutable SUSDS                   = DssExecLib.getChangelogAddress("SUSDS");
+    address internal immutable ILK_REGISTRY            = DssExecLib.getChangelogAddress("ILK_REGISTRY");
+    address internal immutable LITE_PSM                = DssExecLib.getChangelogAddress("MCD_LITE_PSM_USDC_A");
+    address internal immutable MCD_PAUSE_PROXY         = DssExecLib.getChangelogAddress("MCD_PAUSE_PROXY");
```

Comment:
Could we use the [address helper](https://github.com/makerdao/dss-exec-lib/blob/69b658f35d8618272cd139dfc18c5713caf6b96b/src/DssExecLib.sol#L186) as in [checklist](https://github.com/makerdao/pe-checklists/blob/5208017c2c5e04f9ec03817856e06ff9d0fe9ad1/spell/spell-reviewer-mainnet-checklist.md?plain=1#L306)? :)